### ISSUE TITLE
Another weird detection result

### ISSUE THREAD

**Issue by pannal** [guessit-io/guessit#489]
``` 
2017-07-23 04:41:45,789 DEBUG Parsing video: Marvel's Jessica Jones (2015) s01e09 - AKA Sin Bin.mkv, hints: {'type': 'episode'}
2017-07-23 04:41:45,789 INFO Scanning video "Marvel's Jessica Jones (2015) s01e09 - AKA Sin Bin.mkv" in ''
2017-07-23 04:41:45,914 DEBUG GuessIt found: {
    "title": "Marvel's Jessica Jones", 
    "year": 2015, 
    "season": 1, 
    "episode": 9, 
    "language": "Akan", 
    "release_group": "Sin Bin", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

**Comment by ratoaq2** [comment]
This is related to #296 and it's already fixed in an open PR #476:

```
For: Marvel's Jessica Jones (2015) s01e09 - AKA Sin Bin.mkv
GuessIt found: {
    "title": "Marvel's Jessica Jones", 
    "year": 2015, 
    "season": 1, 
    "episode": 9, 
    "episode_title": "AKA Sin Bin", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

I will add this as a regression test in that PR. Thanks

**Comment by ratoaq2** [comment]
Fixed by #476
